### PR TITLE
Client name added to dialog window titles  #1352

### PR DESCRIPTION
**Pull request by dcorson-ticino-com**
routine to make title string in util
4 calls in clientdlg, one for each dialog when it is shown:
Connect, Chat, Musician Profile and Settings

### CODE REVIEW

File: src/clientdlg.cpp [hunk 1]
```diff
@@ -1001,6 +1000,7 @@ void CClientDlg::ShowConnectionSetupDialog()
     // show connect dialog
     bConnectDlgWasShown = true;
     ConnectDlg.show();
+    ConnectDlg.setWindowTitle ( MakeClientNameTitle( tr ( "Connect" ) , pClient->strClientName ) );
```
Comment (pljones) [inline src/clientdlg.cpp:1003]: Sorry, spaces after the method name and before the `(`, too, please, in `MakeClientNameTitle(` - throughout.

File: src/util.cpp [hunk 2]
```diff
@@ -1581,3 +1580,13 @@ QString GetVersionAndNameStr ( const bool bWithHtml )
 
     return strVersionText;
 }
+
+QString MakeClientNameTitle ( QString win, QString client )
```
Comment (pljones) [inline src/util.cpp:1584]: Hm.  I thought we had client name first.  But we don't.  It's "window name clientname" when the client name is specified.  The change to the mixer window was to move the appname to the end -- but _not_ if clientname has been used.  Which is a bit weird but... this is following the same pattern, so fine :).

File: src/clientdlg.cpp (hunk 1 above)
Comment (dcorson-ticino-com) [reply src/clientdlg.cpp:1003]: Fine with me, will do, I was just orienting on the style in the clientdlg constructor.

File: src/clientdlg.cpp [hunk 3]
```diff
@@ -1001,6 +1000,7 @@ void CClientDlg::ShowConnectionSetupDialog()
     // show connect dialog
     bConnectDlgWasShown = true;
     ConnectDlg.show();
+    ConnectDlg.setWindowTitle( MakeClientNameTitle( tr ( "Connect" ) , pClient->strClientName ) );
```
Comment (pljones) [inline src/clientdlg.cpp:1003]: > Sorry, spaces after the method name and before the `(`, too, please

You've removed the one that was before the `(`.  It needs a space both sides.  (Yes, I phrased it a bit clumsily now I read it back.)

File: src/clientdlg.cpp [hunk 4]
```diff
@@ -1000,46 +999,50 @@ void CClientDlg::ShowConnectionSetupDialog()
...
     bConnectDlgWasShown = true;
-    ConnectDlg.show();
+    ConnectDlg.show ();
+    ConnectDlg.setWindowTitle ( MakeClientNameTitle ( tr ( "Connect" ) , pClient->strClientName ) );
 
     // make sure dialog is upfront and has focus
-    ConnectDlg.raise();
-    ConnectDlg.activateWindow();
+    ConnectDlg.raise ();
+    ConnectDlg.activateWindow ();
```
Comment (softins) [inline src/clientdlg.cpp:1007]: I know the formatting style can be confusing! But when submitting a PR, there is usually no need to reformat lines that you haven't changed.

In this case, the "house style" seems to be _no_ space before an empty argument list `()`, but a space before and after an opening `(` when there are arguments to the function.

**Comment by ann0see** [comment]
Any updates here?

**Review by softins — APPROVED** [review]
Compiled and tested ok, and rebased on master to resolve conflicts.

File: src/clientdlg.cpp [hunk 5]
```diff
@@ -948,7 +947,7 @@ void CClientDlg::SetMyWindowTitle ( const int iNumClients )
     {
         // if --clientname is used, the APP_NAME must be the very first word in
         // the title, otherwise some user scripts do not work anymore, see #789
-        strWinTitle += QString ( APP_NAME ) + " " + pClient->strClientName + " ";
+        strWinTitle += QString ( APP_NAME ) + " - " + pClient->strClientName + " ";
```
Comment (pljones) [inline src/clientdlg.cpp:950]: This block (947 to 981) all seems a little clumsy to me and I have to re-read it several times to get the gist.  I think it probably wants a refactor at some point.  Not this change, though.

**Review by pljones — APPROVED** [review]

**Pull request merged**